REVIEW_GUIDELINES.md

```diff
@@ -54,7 +88,7 @@ selector:
   release: {{ .Release.Name }}
 ```
 
-If a chart has multiple components, a `component` label should be added (e. g. `component: server`). The resource name should get the component as suffix (e. g. `name: {{ template "myapp.fullname" . }}-server`). The `component` label must be added to label selectors as well.
+If a chart has multiple components, a `component` label should be added (e. g. `component: server`). The resource name should get the component as suffix (e. g. `name: {{ template "myapp.fullname" . }}-server`).
```

Comment:
Why did you delete this? The component label must be part of the selector.

Reply:
I have separated concerns, so this paragraph does no longer read about selector as there is a dedicated paragraph to it.